src/frontend/components/SchemaForm/_RenderFormInput.tsx

```diff
@@ -70,10 +71,37 @@ export function RenderFormInput(props: IRenderFormInputProps) {
 
   switch (type) {
     case "email":
-    case "password":
     case "url":
     case "color":
       return <FormInput type={type} {...formProps} />;
+    case "password":
+    return (
+        <FormInput
+          type={isPasswordReveal.isOn ? "text" : "password"}
+          {...formProps}
+          rightActions={
+            formProps.input.value
+              ? [
+                  {
+                    systemIcon: "Eye",
```

Comment:
Add this 
```
  EyeOff: `<path d="M17.94 17.94A9.995 9.995 0 0112.009 20H12c-7 0-11-8-11-8a18.605 18.605 0 015.017-5.908l.043-.032M9.9 4.24A8.865 8.865 0 0111.979 4h.023-.001c7 0 11 8 11 8a18.62 18.62 0 01-2.182 3.217l.022-.027m-6.721-1.07a3 3 0 11-4.242-4.238l.002-.002M1 1l22 22" />`,
```
to src/shared/constants/Icons.tsx so that we have a EyeOff icon